KPP/fullchem/CMakeLists.txt

```diff
@@ -61,3 +61,17 @@ target_compile_options(KPP
   $<$<STREQUAL:"${CMAKE_Fortran_COMPILER_ID}","Intel">:-r8>
   $<$<STREQUAL:"${CMAKE_Fortran_COMPILER_ID}","GNU">:-fdefault-real-8 -fdefault-double-8>
 )
+
+# FOR GFORTRAN ONLY: Disable variable tracking for gckpp_Jacobian.F90 in the
+# KPP/fullchem mechanism generated with KPP 3.0.0.  This will avoid a compiler
```

Comment:
Maybe we should remove the version number so that the text doesn't get outdated.